### PR TITLE
Zporeba

### PR DESCRIPTION
**Pull request by ZPoreba**
Routes added.
Dashboard reated.
The navigation bar added as default component on every page.

### CODE REVIEW

File: client/src/Main/MainView.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                  <img alt='intro' src={intro} style={{float: "right", height: "350px"}}/>
+                </div>
+            </QueueAnim>
+          </OverPack>
+          <OverPack style={{ overflow: 'hidden' }} playScale={0.4} >
+            <QueueAnim style={{backgroundColor: "#ffffff"}}>
+                <div key="a" style={{padding: "2%", fontWeight: "bold", fontSize: "x-large"}}>
+                    Contact us
+                </div>
+                <div key="b" style={{paddingLeft: "10%", paddingRight: "10%", display: "flex"}}>
```
Comment (SzymonSmykala) [inline client/src/Main/MainView.js]: Not sure how you separate code to pieces, but this one scary me

File: client/src/routes.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {Route} from 'react-router-dom';
+import LoginView from './Authorization/LoginView';
+import MainView from './Main/MainView';
+import RegistrationView from './Authorization/RegistrationView';
+
+
+const BaseRouter = () => (
+   <div>
+       <Route path="/main" component={MainView} />
```
[the remark this answers is not in the document]
Comment (ZPoreba) [reply client/src/routes.js]: when "/" is called it is redirect to "/main"

File: client/src/Main/MainView.js (hunk 1 above)
Comment (ZPoreba) [reply client/src/Main/MainView.js]: I can separate it into functions

Comment (SzymonSmykala) [reply client/src/Main/MainView.js]: Do It. Keep the code clean.

Comment (ZPoreba) [reply client/src/Main/MainView.js]: Done

**Pull request merged**